src/i18n/i18n_module_en.properties

```diff
@@ -2396,3 +2396,54 @@ ownership=Ownership
 validation_result=Validation result
 sql=SQL
 categories_cannot_be_empty=Categories cannot be empty
+
+# Translations for shortcuts and app title in the manifest file
+MANIFEST_APP_TITLE=Maintenance app
```

Comment:
in pot files, you can add a context to describe where the string is used - this is not possible in properties file (so translators might not know the context of the string (unless we communicate that it is prefixed by MANIFEST_)